SpatialGDK/Source/SpatialGDK/Private/Interop/ClientConnectionManager.cpp

```diff
@@ -76,6 +77,30 @@ void ClientConnectionManager::OnRequestReceived(const Worker_Op&, const Worker_C
 void ClientConnectionManager::RegisterClientConnection(const Worker_EntityId InWorkerEntityId, USpatialNetConnection* ClientConnection)
 {
 	WorkerConnections.Add(InWorkerEntityId, ClientConnection);
+
```

Comment:
This section is in this PR also - https://github.com/spatialos/UnrealGDK/pull/3286/files.
Reminder to close that other PR when this goes in.

Reply:
Well pointed out. Will do!